STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/ErrorMappers/ErrorMapperServiceTests.Logic.Retrieve.cs

```diff
@@ -0,0 +1,45 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using STX.REST.RESTFulSense.Clients.Models.Errors;
+using Tynamix.ObjectFiller;
+using Xunit;
+
+namespace STX.REST.RESTFulSense.Clients.Tests.Unit.Services.Foundations.ErrorMappers
+{
+    public partial class ErrorMapperServiceTests
+    {
+        [Fact]
+        private async Task ShouldRetrieveStatusDetailByStatusCodeAsync()
```

Comment:
I wonder if this could be a Theory with an input parameter for each valid status code accepted.